crates/driver/src/boundary/liquidity/mod.rs

```diff
@@ -38,7 +38,7 @@ const BLOCK_POLL_INTERVAL: Duration = Duration::from_secs(1);
 fn cache_config() -> CacheConfig {
     CacheConfig {
         number_of_blocks_to_cache: NonZeroU64::new(10).unwrap(),
-        number_of_entries_to_auto_update: NonZeroUsize::new(200).unwrap(),
+        number_of_entries_to_auto_update: NonZeroUsize::new(1000).unwrap(),
```

Comment:
Increased the threshold because we'd like to update **ALL** liquidity sources in the background so we ideally only have to read from the cache when a new auction appears.
Eventually we should make these values configurable.

Reply:
Do these need to be chunked as well (given you decided to use 200 entry chunks to cope with 1400 total requests)

Reply:
The main reason we need chunking for the regular `.get()` function is that it's possible to issue so many liquidity fetch requests when starting the backend that it doesn't fit in the 15s deadline. If we don't chunk in that case we'll never end up caching any data.
When testing this change against the prod mainnet load this did not seem necessary for the background task updating the liquidity.
I would leave it like this for the next test but in the future we can certainly consider chunking that as well.